documentation/modules/overview/con-overview-components-topic-operator.adoc

```diff
@@ -6,29 +6,30 @@
 [id='overview-concepts-topic-operator-{context}']
 = Topic Operator
 
-The Topic Operator provides a way of managing topics in a Kafka cluster through Kubernetes resources.
+[role="_abstract"]
+The Topic Operator provides a way of managing topics in a Kafka cluster through `KafkaTopic` resources.
 
 .Example architecture for the Topic Operator
 
 image:topic-operator.png[The Topic Operator manages topics for a Kafka cluster via KafkaTopic resources]
 
-The role of the Topic Operator is to keep a set of `KafkaTopic` Kubernetes resources describing Kafka topics in-sync with corresponding Kafka topics.
+The Topic Operator manages Kafka topics by watching for `KafkaTopic` resources that describe Kafka topics, and ensuring that they are configured properly in the Kafka cluster.
 
-Specifically, if a `KafkaTopic` is:
+When a `KafkaTopic` is created, deleted, or changed, the Topic Operator performs the corresponding action on the Kafka topic.
 
-* Created, the Topic Operator creates the topic
-* Deleted, the Topic Operator deletes the topic
-* Changed, the Topic Operator updates the topic
+You can declare a `KafkaTopic` as part of your application's deployment and the Topic Operator manages the Kafka topic for you.
+
+The Topic Operator maintains information about each topic in a _topic store_, which is continually synchronized with updates from Kubernetes `KafkaTopic` custom resources or Kafka topics.
+Updates from operations applied to a local in-memory topic store are persisted to a backup topic store on disk.
+
+The Topic Operator operates in the following modes: 
+
+Bidirectional mode:: Bidirectional mode means that the Topic Operator can reconcile changes to a `KafkaTopic` resource to and from a Kafka cluster.
+This means that you can update topics either through the `KafkaTopic` resource or directly in Kafka, and the Topic Operator will ensure that both sources are updated to reflect the changes. This mode can be used with ZooKeeper cluster management. 
+Unidirectional mode (preview):: Unidirectional means that the Topic Operator solely manages topics through the `KafkaTopic` resource. This mode can be used with KRaft cluster management.
+
+NOTE: Unidirectional mode and KRaft cluster management are available to try by enabling the `UnidirectionalTopicOperator` and `UseKRaft` feature gates. 
```

Comment:
I do not think Tom's PR enables this. So this would need an additional follow-up PR. So I wonder if we want to exclude it for the time being to avoid having it merged by mistake.

Reply:
Right. I've removed from a  couple of places that suggest the support is there in the preview.